WG-TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# OCI Working Group Template
+
+The following is a template that the TOB and/or proposers of a new OCI Working Group may use to document the intended scope and activities of the OCI Working Group following its establishment. {Bracketed sections below} should be filled in with the Working Group's specific details.
+
+# {WG NAME} OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+{Briefly describe the purpose of the WG.}
+
+## Scope
+
+* {Describe the activities and work that are in scope for the WG.}
+
+## Out of Scope
+
+* {Where helpful for clarity or to avoid confusion, describe any activities or topics that are out of scope for the WG.}
+
+## Intended work product
+
+* {Describe the intended work product and outputs of the WG, particularly indicating whether they consist of specification(s) and/or project(s).}
+
+## Governance
+
+* **Working Group**:
+  * The TOB is establishing the WG as an OCI Working Group, pursuant to section 6(p) of the OCI Charter.
+* **Maintainers**:
+  * The TOB will appoint one or more initial maintainers of the WG.
+  * The TOB may, in its discretion, remove and/or appoint new maintainers of the WG.
+  * The current maintainers will be listed in the [OCI Working Group documentation](https://github.com/opencontainers/tob/blob/master/WG-INFO.md).
+  * The maintainers shall be responsible for:
+    * scheduling regular meetings of the WG community;
+    * facilitating open discussion among WG community participants;
+    * coordinating and managing the development of the WG work product and outputs;
+    * recording decisions that are reached by the WG community; and
+    * keeping the TOB regularly informed about the status of the WG’s efforts, including when the WG has readied the work product and outputs for TOB approval.
+* **Meetings**:
+  * Meetings of the WG shall be open to the public.
+  * Participants in the meetings shall comply with the [OCI Code of Conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md) and all other policies of OCI and The Linux Foundation.
+* **TOB Approval**:
+  * The WG shall operate pursuant to the procedures set forth in section 6(p) of the OCI Charter, with regards to obtaining TOB approval for initial release of the work product and outputs as an OCI Specification or OCI Project, and for subsequent maintenance activities thereafter.
```

Comment:
For these links to sections, I'm wondering if they should be smaller headings so we can link directly to them? if this is of interest I can PR to do this (and update the links here) after this is merged.

Reply:
That sounds like a good idea -- I'll defer to others but that sounds great to me.